xen/arch/arm/coproc/schedule.c

```diff
@@ -201,26 +201,30 @@ static inline void schedule_trace(struct vcoproc_instance *curr,
         break;
 
     case 1:
-        COPROC_VERBOSE(NULL, "--dom %d (%s) CONTINUE RUNNING (BUSY)----\n",
+        COPROC_VERBOSE(curr ? curr->mcoproc->dev : NULL,
+                       "--dom %d (%"PRIpaddr") CONTINUE RUNNING (BUSY)----\n",
                        curr ? curr->domain->domain_id : -1,
-                       curr ? dev_path(curr->coproc->dev) : "NULL");
+                       curr ? curr->mmios[0].addr : 0);
```

Comment:
Because, mmios[0] is expected to be present for any vcoproc and expected to be unique for within a domain for vcoprocs running on one mcoproc.